src/regex.ts

```diff
@@ -110,8 +110,12 @@ const regexStringFromPattern: (pattern: Pattern) => string = (pattern) => {
 export const regexFromPattern = (
 	pattern: Pattern,
 	caseInsensitive = false,
+	global = false,
+	multiline = false,
```

Comment:
this should probably be an object, but in the interest of not requiring a major version bump this is fine